examples/common/builders.py

```diff
@@ -39,6 +40,9 @@ def build_callback(name, kwargs):
             'log_optimizer_metrics', True),)
     elif name == 'health_checker':
         return HealthChecker(**kwargs)
+    elif name in RESUMPTION_STRATEGIES:
```

Comment:
```suggestion
    elif name in RESUMPTION_STRATEGIES.keys():
```

makes it explicit that this is a dict... idk you can ignore if you want.